cr-core/src/main/java/org/terasology/crashreporter/logic/GoogleDriveConnector.java

```diff
@@ -175,7 +177,7 @@ private static Credential authorize(HttpTransport httpTransport) throws GeneralS
     }
 
     private static PrivateKey getPrivateKey() throws IOException, GeneralSecurityException {
-        URL website = new URL("http://jenkins.terasology.org/userContent/GooeyDrive-24506b2f1f34.p12");
+        URL website = new URL("https://jenkins.terasology.io/cjoc/userContent/gooey-drive-371f4f7d4d08.p12");
```

Comment:
Can/should we put a docstring here explaining where this file comes from/how to recreate in case it gets lost or we want to rotate it? For instance, a link to the cloud cockpit for the respective account - or is this too risky/exposing too much information?